packages/babel-plugin-proposal-decorators/test/fixtures/legacy-object-ordering/options.json

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["env"],
+  "presets": [["env", { "targets": { "browsers": "ie 6" } }]],
```

Comment:
Really a nit and only if you can do it with a regexp, but I find the top-level `targets` _slightly_ more readable (less `[` and `{`).

Reply:
Do we still want to make this change? Guess it could be a separate PR?

Reply:
We could track this in a separate PR but if we use top level targets, the output fixtures may need to be updated since plugins can be aware of that. This PR aims for minimizing fixtures update so we can move forward.

Reply:
Ok yeah that sounds like a different issue entirely given that chance.

Really want to work on making these diffs easier to look through! Would be interesting to run our tests through a combination of retainLines/recast